DangoPlop/Assets/Scripts/SceneSelection.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class SceneSelection : MonoBehaviour {
```

Comment:
I'm assuming this script was attached to a new GameObject which uses the SceneSelection prefab right?